docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -257,6 +259,7 @@ Update the configuration:
 $ tctl create -f cap.yaml
 cluster auth preference has been updated
 ```
+
```

Comment:
Good opportunity to use the new `tctl edit` feature:

```suggestion

You can also edit this configuration directly:

```code
$ tctl edit cluster_auth_preference
```

```

Reply:
Thanks for this, I didn't knew we have `tctl edit` command.